Documentation.md

```diff
@@ -906,128 +941,95 @@ currently this is limited to softphone-enabled voice contacts.
-```
-conn.hold({
-   success: function() { ... },
-   failure: function() { ... }
-});
-```
-Put this voice connection on hold.
-
-### `voiceConnection.resume()`
-```
-conn.resume({
-   success: function() { ... },
-   failure: function() { ... }
-});
-```
-Only has an effect if this connection was on hold when called.
-
-### `voiceConnection.isOnHold()`
-```
-if (conn.isOnHold()) { ... }
-```
-Returns true if this connection is on hold, false otherwise.
-
 ### `voiceConnection.getMediaInfo()`
-```
-const mediaInfo = conn.getMediaInfo();
+```js
+var mediaInfo = conn.getMediaInfo();
 ```
 Returns the media info object associated with this connection.
 
 ### `voiceConnection.getMediaType()`
+```js
+if (conn.getMediaType() === "softphone") { ... }
 ```
-if (conn.getMediaType()==="SOFTPHONE") { ... }
-```
-Returns the softphone media type enum: `"SOFTPHONE"`.
+Returns the `MediaType` enum value: `"softphone"`.
 
 ### `voiceConnection.getMediaController`
-```
-const voiceController = conn.getMediaController();
+```js
+var voiceController = conn.getMediaController();
 ```
 Get the media controller associated with this connection.
 
 ## ChatConnection API
 The ChatConnection API provides action methods (no event subscriptions) which can be called to manipulate the state
-of a particular chat connection within a contact.  Like contacts, connections come and go. It is good practice not
+of a particular chat connection within a contact. Like contacts, connections come and go. It is good practice not
 to persist these object or keep them as internal state. If you need to, store the `contactId` and `connectionId`
 of the connection and make sure that the contact and connection still exist by fetching them in order from
 the `Agent` API object before calling methods on them.
 
 ### `chatConnection.getMediaInfo()`
-```
-const mediaInfo = conn.getMediaInfo();
+```js
+var mediaInfo = conn.getMediaInfo();
 ```
 Get the media info object associated with this connection.
 
 ### `chatConnection.getConnectionToken()`
-```
+```js
 conn.getConnectionToken()
-  .then(response => {})
-  .catch(error => {});
+  .then(function(response) { /* ... */ })
+  .catch(function(error) { /* ... */ });
 ```
-Provides a promise which resolves with the API response from createTransport transportType chat_token for this connection through the createTransport LARS CTI API.
+Provides a promise which resolves with the API response from createTransport transportType chat_token for this connection.
 
 ### `chatConnection.getMediaType()`
+```js
+if (conn.getMediaType() === "chat") { /* ... */ }
 ```
-if (conn.getMediaType()==="CHAT") { ... }
-```
-Returns mediaType for a chat connection, which is `"CHAT"`.
+Returns the `MediaType` enum value: `"chat"`.
 
 ### `chatConnection.getMediaController()`
-```
-const mediaController = conn.getMediaController();
+```js
+var mediaController = conn.getMediaController();
 ```
 Get the media controller associated with this connection.
```

Comment:
After writing the TypeScript for this, I'm wondering whether this should be a public API...